docs/source/en/api/pipelines/mochi.md

```diff
@@ -15,15 +15,59 @@
 
 # Mochi
 
-[Mochi 1 Preview](https://huggingface.co/genmo/mochi-1-preview) from Genmo.
+> [!TIP]
+> Only a research preview of the model weights is available at the moment.
+
+[Mochi 1](https://huggingface.co/genmo/mochi-1-preview) is a video generation model by Genmo with a strong focus on prompt adherence and motion quality. The model features a 10B parameter Asmmetric Diffusion Transformer (AsymmDiT) architecture, and uses non-square QKV and output projection layers to reduce inference memory requirements. A single T5-XXL model is used to encode prompts.
 
 *Mochi 1 preview is an open state-of-the-art video generation model with high-fidelity motion and strong prompt adherence in preliminary evaluation. This model dramatically closes the gap between closed and open video generation systems. The model is released under a permissive Apache 2.0 license.*
 
-<Tip>
+> [!TIP]
+> Make sure to check out the Schedulers [guide](../../using-diffusers/schedulers) to learn how to explore the tradeoff between scheduler speed and quality, and see the [reuse components across pipelines](../../using-diffusers/loading#reuse-a-pipeline) section to learn how to efficiently load the same components into multiple pipelines.
+
+## Quantization
+
+Quantization helps reduce the memory requirements of very large models by storing model weights in a lower precision data type. Refer to the [Quantization](../../quantization/overview) to learn more about supported quantization backends and selecting a quantization backend that supports your use case.
```

Comment:
Hmm, I am thinking if this makes sense to only show `bitsandbytes` now that we have another quantization backend: TorchAO. We will soon have GGUF, too.

Reply:
Maybe I can better clarify that bitsandbytes is just an example and there are other supported quantization backends users can pick from at https://moon-ci-docs.huggingface.co/docs/diffusers/pr_10249/en/quantization/overview ?

I think if we also show torchao/gguf, it could get a bit heavy.

Reply:
That works for me too.